director/urls_tests.py

```diff
@@ -215,7 +219,7 @@ def test_urls(self):
             content = response.content.decode("utf-8")
             for expect in expects if isinstance(expects, list) else [expects]:
                 if isinstance(expect, int):
-                    assert response.status_code == expect
+                    assert response.status_code == expect, check.path
```

Comment:
This shows the path being checked in the output which makes it a little bit easier to see what's going on.